File: 1주차/matl3rayc7/reverse_words_in_a_string.py
class Solution:
    def reverseWords(self, s: str) -> str:
        x = s.split()
        str = ''
        for i in range(0, len(x)):
            if i != len(x) -1:
                str += x[len(x)-1-i] + ' '
            else:
                str += x[len(x)-1-i]
        return str
        

File: 1주차/mijip0320/reverse_words_in_a_string.js
var reverseWords = function (s) {
    let revStr = "";

    //split s with blanks and reverse, and filter that are not blanks
    const splitString = s.split(' ').reverse().filter((el) => el !== "");
  
    for (let i = 0; i < splitString.length; i++){
        if (i !== 0) {
            //when it is the second word, add with a blank
            revStr += ' ' + splitString[i];

        } else {
            //when it is the first word, just add the word
            revStr += splitString[0]
        }
    }

    console.log(revStr)
};
